api/app/src/mappings/google/biometrics.ts

```diff
@@ -62,23 +59,23 @@ export const mapToBiometrics = (googleBiometrics: GoogleBiometrics, date: string
           spo2: {
             avg_pct: Util.getAvgOfArr(values),
             max_pct: min_max.max_item,
-            min_pct: min_max.min_item
-          }
-        }
+            min_pct: min_max.min_item,
+          },
+        };
       }
 
-      if (data.dataSourceId = sourceIdHeartBpm) {
+      if (data.dataSourceId === sourceIdHeartBpm) {
```

Comment:
@Orta21 just to make sure, here we want to compare, not assign, right?

Reply:
^ good catch